operator/duties/scheduler_test.go

```diff
@@ -237,3 +237,46 @@ func TestScheduler_Run(t *testing.T) {
 	cancel()
 	require.NoError(t, s.Wait())
 }
+
+func TestScheduler_Regression_IndiciesChangeStuck(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
```

Comment:
In general this PR should be related to "Refactored Monitoring guide", please next time open a separate PR for separate tasks.

Reply:
Ah sorry missed your comment above, no prob :)